src/engine-actions/get-channels-for-address.js

```diff
@@ -0,0 +1,34 @@
+const { networkAddressFormatter } = require('../utils')
+const { listChannels, listPendingChannels } = require('../lnd-actions')
+
+/**
+ * Get all channels to the given address
+ * @param {string} address
+ * @returns {Array} channels
+ */
+async function getChannelsForAddress (address) {
```

Comment:
The assumption in this method is that we are searching for a remote address. Is it overkill to have this be `getChannelsForRemoteAddress`?